src/frontend/src/customNodes/genericNode/components/parameterComponent/index.tsx

```diff
@@ -257,6 +265,44 @@ export default function ParameterComponent({
     );
   }, [info]);
 
+  function renderTitle() {
+    const output_types = title.split("|");
+    const displayTitle = data.selected_output_type ?? output_types[0];
+    return !left && output_types.length > 1 ? (
+      <DropdownMenu>
```

Comment:
I believe it would enhance readability to create a separate component for this <Dropdown>. The parameterComponent is already big enough, wouldn't you agree?
@lucaseduoli